src/CMakeLists.txt

```diff
@@ -523,6 +523,47 @@ SET(OpenMVG_INCLUDE_DIRS
 
 INCLUDE_DIRECTORIES(${OpenMVG_INCLUDE_DIRS})
 
+find_package(Popsift)
+if(POPSIFT_FOUND)
+  ADD_DEFINITIONS(-DHAVE_POPSIFT)
+  INCLUDE_DIRECTORIES(${POPSIFT_INCLUDE_DIR})
+  LINK_DIRECTORIES(${POPSIFT_LIBRARY_DIR})
+endif()
+
+find_package(CUDA 7.0) #use CUDA 7.0 because of nvcc bug with CUB
```

Comment:
[note for later] not necessary (with the modern cmake version of PopSift, master/develop one). Cuda is a secondary dependences imported directly by PopSift.
we will just need
```cmake
find_package(PopSift CONFIG)
if(POPSIFT_FOUND)
   ADD_DEFINITIONS(-DHAVE_POPSIFT)
   INCLUDE_DIRECTORIES(${POPSIFT_INCLUDE_DIR})
   LINK_DIRECTORIES(${POPSIFT_LIBRARY_DIR})
 endif()
```